lib/oplog/oplogEntry.go

```diff
@@ -148,6 +130,23 @@ func (op *oplogEntry) ChangedFields() []string {
 	return []string{}
 }
 
+// Given a bson.Raw object, returns the top level keys of the document it represents
+func mapKeysRaw(rawData bson.Raw) []string {
```

Comment:
I know that many of the other tests also cover this code, but it would be nice to have a couple of unit tests for this function specifically.

Reply:
Good idea